repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -75,6 +76,7 @@ import scala.io.Source
   */
 class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMode: ResponseMode, bodyMode: ResponseMode, desiredOutputStream: ServletOutputStream)
   extends javax.servlet.http.HttpServletResponseWrapper(originalResponse) with HeaderInteractor {
+  private val addHeaderWarningLogger = LoggerFactory.getLogger(this.getClass.getName + "_addHeaderWarning")
```

Comment:
I wonder if we actually want to use `this.getClass.getName`. We could just use the static `classOf[HttpServletResponseWrapper].getName`. Obviously that means that extensions of this class will have the same named logger, but that's probably a good thing. It would be kind of annoying to have to manage multiple loggers to turn off or reroute these log messages. Also, it doesn't cost much, but `this.getClass` is slightly less performant than `classOf[HttpServletResponseWrapper]`.